### PR TITLE
Issue #129 - Reduce storage size

### PR DESCRIPTION
**Pull request by pecko**

### CODE REVIEW

File: info.limpet/src/info/limpet/data/persistence/xml/XStreamHandler.java [hunk 1]
```diff
@@ -62,7 +62,14 @@
 		
 		
 		xstream.alias("Point", org.geotools.geometry.iso.primitive.PointImpl.class);
```
Comment (IanMayo) [inline info.limpet/src/info/limpet/data/persistence/xml/XStreamHandler.java:64]: @pecko - for the Point handling, I hope we can just store [x,y] coordinate pairs.  We can use our GeoSupport class to convert them to/from PointImpl objects.  Then we won't need any of these other geospatial metadata fields.

File: info.limpet/src/info/limpet/data/impl/LimpetList.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package info.limpet.data.impl;
+
+import java.util.ArrayList;
+
```
Comment (IanMayo) [inline info.limpet/src/info/limpet/data/impl/LimpetList.java]: @pecko  - could you put in a class level comment explaining why we're extending ArrayList?

Comment (pecko) [inline info.limpet/src/info/limpet/data/impl/LimpetList.java]: Yes, I will do it during day.

File: info.limpet/src/info/limpet/data/persistence/xml/XStreamHandler.java (hunk 1 above)
Comment (pecko) [inline info.limpet/src/info/limpet/data/persistence/xml/XStreamHandler.java:64]: I think that.

**Pull request merged**